pymatviz/ptable.py

```diff
@@ -700,291 +765,352 @@ def histogram(
 
 
 def ptable_heatmap(
-    values: ElemValues,
+    data: pd.DataFrame | pd.Series | dict[str, list[list[float]]],
     *,
-    log: bool | Normalize = False,
-    ax: plt.Axes | None = None,
-    count_mode: ElemCountMode = ElemCountMode.composition,
-    cbar_title: str = "Element Count",
-    cbar_range: tuple[float | None, float | None] | None = None,
+    # Heatmap specific
+    colormap: str = "viridis",
+    log: bool = False,
+    sci_notation: bool = False,
```

Comment:
I think we might just make our life easier to add this `sci_notation` arg to manually toggle scientific notation mode, instead of trying to guess whether we need scientific notation mode or not (because I cannot think about a deterministic and algorithmic way to decide this).